src/models/application_configuration.js

```diff
@@ -47,6 +51,9 @@ function parse (config, value) {
     case 'inverted-bool': {
       return (value === 'false');
     }
+    case 'force-https': {
+      return (value === false) ? 'DISABLED' : 'ENABLED';
```

Comment:
```suggestion
      return (value === 'false') ? 'DISABLED' : 'ENABLED';
```

Here's what I get in my test:

```
clever config set force-https false
Force redirection of HTTP to HTTPS: enabled
```